### ISSUE TITLE
[bug]: A lot of XHR 404 in console

### ISSUE THREAD

**Issue by AbsurdlySuspicious** [makeplane/plane#3615]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

A lot of 404 errors all with endpoints `/1/decide` or `/1/e` in console:
![image](https://github.com/makeplane/plane/assets/46577212/02c06061-1d14-47c0-a030-ebc394acdd39)

In network tab, all these requests are redirected from same request with trailing slash:
![image](https://github.com/makeplane/plane/assets/46577212/313d5401-4dc9-4b75-ad04-c2e32624c8cc)

I'm not sure if this is related, but dynamic update is also completely missing: no notifications or page updates until refresh. May be related to #3609 

### Steps to reproduce

1. Setup clean v0.15 or v0.14.2 release from docker hub (didn't test previous versions)
2. Setup admin user on /god-mode
3. Exit god mode
4. Open console

### Browser

Mozilla Firefox

### Version

Self-hosted

**Comment by 1akhanBaheti** [comment]
Hello @AbsurdlySuspicious it seems for posthog initialization, an incorrect host is provided.
can you check that once?

**Comment by quocln** [comment]
Hello @AbsurdlySuspicious,

Add these variables in .env

`POSTHOG_API_KEY=<your key>
POSTHOG_HOST=https://app.posthog.com`

See more detail here:
https://posthog.com/docs/libraries/js

**Comment by AbsurdlySuspicious** [comment]
Yep, that was due to posthog. Thank you for pointing me in the right direction 

Since we are not using it, we had to resort to removing posthog wrapper from web/lib/app-provider.tsx 
It would be nice if there was an option to disable it on client through configuration or env vars
